exports/exports.go

```diff
@@ -226,3 +235,88 @@ func (e *Export) getExportWithUser(w http.ResponseWriter, r *http.Request) *mode
 
 	return export
 }
+
+func DBExportToAPI(payload models.ExportPayload) ExportPayload {
+	apiPayload := ExportPayload{
+		ID:          payload.ID.String(),
+		CreatedAt:   payload.CreatedAt,
+		CompletedAt: payload.CompletedAt,
+		Expires:     payload.Expires,
+		Name:        payload.Name,
+		Format:      string(payload.Format),
+		Status:      string(payload.Status),
+	}
+	for _, source := range payload.Sources {
+		newSource := Source{
+			ID:          source.ID,
+			Application: source.Application,
+			Status:      string(source.Status),
+			Resource:    source.Resource,
+			Filters:     source.Filters,
+		}
+
+		if source.SourceError != nil {
+			newSource.Message = &source.SourceError.Message
+			newSource.Code = &source.SourceError.Code
+		}
+
+		apiPayload.Sources = append(apiPayload.Sources, newSource)
+	}
+
+	return apiPayload
+}
+
+func APIExportToDBExport(apiPayload ExportPayload) (*models.ExportPayload, error) {
+	payload := models.ExportPayload{
+		CreatedAt: apiPayload.CreatedAt,
+		Expires:   apiPayload.Expires,
+		Name:      apiPayload.Name,
+	}
+
+	// use the ID from the request if it's present
+	if apiPayload.ID != "" {
```

Comment:
We need to think about this a bit.  I kind of think we should let the db layer create the uuid when it needs to (when the data is inserted into the table would be ideal, i think).

Currently the way this works, it would allow a caller to pass in the uuid that gets set in the database.  I don't think that's really what we want.